### PR TITLE
Update quickhelp.js

### PR DESCRIPTION
**Pull request by abalkin**
Use left/right instead of before/after.  See #7807.

### CODE REVIEW

**Comment by Carreau** [comment]
Just to be annoying, how does that behave with arabic, or other RTL dialect :-P

**Comment by Carreau** [comment]
And to respond to myself: 

http://codemirror.net/demo/bidi.html

It is indeed after/before, and not left/right, still the behavior can be configured. 

Though I guess if using RTL lang, we can assume things are customized.

**Comment by takluyver** [comment]
Yeah, in this context, it really is the word before/after the cursor, not left/right. Whereas moving through the text with arrow keys goes in the direction the arrow points, so left/right is appropriate. So I don't think this change makes sense.

+1 to changing 'del' to 'delete', though - do you want to amend this commit to just do that?

**Comment by abalkin** [comment]
Moreover, in RTL locale, you would have to change the pictograms:

screen shot 2015-02-17 at 8 22 21 pm

**Comment by Carreau** [comment]
I'm not sure the pictogram change on RTL, they are still in this direction on keyboard. 
The arrow to the left does always delete Before cursor whatever direction that is.
As in exist sign stickyman can run to the right even if actual dors is on the left.

**Comment by takluyver** [comment]
Well, it [does exist](http://www.fileformat.info/info/unicode/char/2326/index.htm), but we're not about to build logic into the shortcuts dialog to determine whether the cursor is in RTL text. :-)

**Comment by abalkin** [comment]
> The arrow to the left does always delete Before cursor whatever direction that is.

Not really, see http://support.apple.com/en-us/HT201794

screen shot 2015-02-17 at 8 29 52 pm

**Comment by Carreau** [comment]
Hum, I guess I've only seen seen windows-arabic then. 
Joy of internationalisation.

**Comment by abalkin** [comment]
From the Apple Style Guide (https://help.apple.com/asg/mac/2013/ASG_2013.pdf):

screen shot 2015-02-17 at 8 42 05 pm

**Comment by takluyver** [comment]
i.e. even apple is inconsistent between 'Forward' and 'Right'. I would guess that in RTL text it deletes the character after (i.e. to the left of) the cursor.

File: IPython/html/static/notebook/js/quickhelp.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ define([
             { shortcut: "Ctrl-Down",  help:"go to cell end"  },
             { shortcut: "Ctrl-Left",  help:"go one word left"  },
             { shortcut: "Ctrl-Right", help:"go one word right"  },
-            { shortcut: "Ctrl-Backspace", help:"del word before"  },
-            { shortcut: "Ctrl-Delete",    help:"del word after"  },
```
Comment (takluyver) [inline IPython/html/static/notebook/js/quickhelp.js]: and these ones ;-)

**Comment by abalkin** [comment]
Github makes it amazingly easy to edit files.  Is rebasing easy as well?

**Comment by takluyver** [comment]
I don't think so, sadly. But I think for a change this small, it's not worth your time to rebase it.

**Pull request merged**